Simulator/Recorders/Hdf5Recorder.h

```diff
@@ -70,6 +70,8 @@ class Hdf5Recorder : public IRecorder {
    /// Registered to OperationManager as Operation::printParameters
    virtual void printParameters() override;
 
+   virtual void registerVariables(std::string varName, EventBuffer &recordVar);
```

Comment:
Since this is just registering one variable at a time, I think it could be called `registerVariable()`

Reply:
Also, should this be declared `override`?

Reply:
Yes, I should do that.